src/gluonts/ext/r_forecast/_univariate_predictor.py

```diff
@@ -136,7 +137,29 @@ def _get_r_forecast(self, data: Dict) -> Dict:
         r_params = self._robjects.vectors.ListVector(self.params)
         vec = self._robjects.FloatVector(data["target"])
         ts = make_ts(vec, frequency=self.period)
-        forecast = self._r_method(ts, r_params)
+
+        if (
+            "feat_dynamic_real" in data
+            and self.method_name == "fourier.arima.xreg"
+        ):
+            import rpy2.robjects.numpy2ri
+
+            rpy2.robjects.numpy2ri.activate()
```

Comment:
Also for this one: I’m not sure what this does, but is this something that needs to happen for every forecast that is generated?

Reply:
This is to enable conversion from numpy array ('feat_dynamic_real') to r matrix. https://rpy2.github.io/doc/v2.9.x/html/numpy.html#from-numpy-to-rpy2:~:text=From%20numpy%20to%20rpy2%3A%C2%B6

Reply:
Yes, this will be repeated for every item/forecast by keeping the activate() at this location.